server/i18n/en.json

```diff
@@ -8252,7 +8252,15 @@
   },
   {
     "id": "ent.outgoing_oauth_connections.authenticate.app_error",
-    "translation": "There was an error while authenticating the outgoing oauth connection."
+    "translation": "There was an error while authenticating the outgoing oauth connection: {{ .Error }}"
+  },
+  {
+    "id": "ent.outgoing_oauth_connections.connection_matching_audience_exists.app_error",
+    "translation": "There is already an outgoing oauth connection for the provided audience."
+  },
+  {
+    "id": "ent.outgoing_oauth_connections.connection_matching_audience_exists.not_found",
+    "translation": "No configured outgoing oauth connection matches the provided audience."
```

Comment:
Just to be sure, is this error ever shown when it's "okay" that a URL did not match a configured audience? i.e. I'm setting up a slash command that purposely does not use any configured OAuth connections.

Reply:
That error is only being used to check if a connection with a provided audience already exists.